firefox-ios/Client/Glean/tags.yaml

```
@@ -60,6 +60,9 @@ Passwords:
 PasswordGenerator:
   description: Corresponds to the generation of strong passwords for logins and autofill.
 
+ReaderMode:
+  description: TODO: Add description for ReaderMode tag
```

```suggestion
  description: Corresponds to the reading mode viewer for webpages.
```
As discussed, thanks for checking in the [ticket](https://github.com/mozilla-mobile/firefox-ios/issues/27326#issuecomment-2993024045). 😄